style.css

```diff
@@ -110,30 +114,44 @@ h2 {
 .year-book {
   text-align: center;
   align-self: center;
-  margin: 20px 12px;
+  margin: 20px 12px 5px;
+}
+
+.year-book {

```

Comment:
would you like to add this to one single class?

Reply:
yes, I see the problem here. As we talk I will create a new class to wrap both elements in one rule to affect them and leave the class "year-book" with his own margin.